catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -187,9 +185,12 @@ private void combineXpathFilterQueries(SolrQuery query, List<SolrQuery> subQueri
             for (String xpath : xpathIndexes) {
                 indexes.add("(" + XPATH_FILTER_QUERY_INDEX + ":\"" + xpath + "\")");
             }
-            String index = XPATH_QUERY_PARSER_PREFIX + StringUtils.join(indexes, operator);
```

Comment:
If I'm reading this right, are you fixing a bug here? It looks like before we were doing the index query logic even if there were no indexes in the query? Or did this only break with the Solr 5 upgrade and it was fine before?